src/app/static/src/app/main.ts

```diff
@@ -9,3 +9,5 @@ export const store = new Vuex.Store<RootState>(storeOptions);
 registerTranslations(store);
 
 export const inactiveFeatures: string[] = [];
+
+store.dispatch({type: "projects/getCurrentProject"}, {root: true});
```

Comment:
Huh, this actually makes a lot fo sense here, but all other "start-up" actions we have put in `index.ts` in the app itself (and tested that these are called from `app.test.ts`) so I think for consistency, this should go there, and a test for this added.

This also seems to be causing issues in the tests - I'm seeing a lot of console error output in the tests on Travis, presumably because this is being called and not mocked.Try moving it and see if it helps with your test failure.